src/main/java/com/springweb/dv_spring_web_mongo/dto/ProjectDTO.java

```diff
@@ -14,9 +15,12 @@ public ProjectDTO(String id, String projectName) {
         this.projectName = projectName;
     }
 
+
+    @NotNull
```

Comment:
Now I cannot create or update a project without specifying the `id` property of body. But in fact the field isn't used on backend side. The `id` field souldn't be required, or maybe it is better to intrioduce new dto object for correspond methods without `id` field?